docs/src/install/core.rst

```diff
@@ -21,9 +26,10 @@ User's recommended
 
```

Comment:
Maybe rename this section to "Recommended to all users" and next one to smth like "Recommended to users who participate in development of Orion"

Reply:
> his way is recommended if you want to work with the bleeding edge version of Oríon.

Also, below it turns out that `master` branch is not bleeding edge? Why is this recommended then?